index.bs

```diff
@@ -737,6 +769,51 @@ authorizing an authenticator.
 </div>
 
 
+#### Construct the credential - `constructCredentialCallback` method #### {#constructCredentialCallback}
+
+This method's purpose is to map |credentialCreationData|'s [=items=] into a new {{PublicKeyCredential}} object associated
+with the caller's [=environment settings object/global object=], while running on the main [=event loop=]. See also
+[[!CREDENTIAL-MANAGEMENT-1]].
+
+This algorithm accepts one argument:
+
+    :   |global|
+    ::  Which must be its caller's [=current settings object=]'s [=environment settings object/global object=].
+
+When this method is invoked, the user agent MUST execute the following algorithm:
+
+1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
+    bytes of |credentialCreationData|.<code>[=attestationObjectResult=]</code>'s value.
+
+3.  Let |id| be |attestationObject|.<code>authData.attestation data.credential ID</code> (see [[#sec-attestation-data]]
+    and [[#sec-authenticator-data]]).
+
+4.  Let |pubKeyCred| be a new {{PublicKeyCredential}} object associated with |global| whose fields are:
+
+    :   {{PublicKeyCredential/[[identifier]]}}
+    ::  |id|
+
+    :   {{PublicKeyCredential/response}}
+    ::  A new {{AuthenticatorAttestationResponse}} object associated with |global| whose fields are:
+
+        :   {{AuthenticatorResponse/clientDataJSON}}
+        ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of
+            |credentialCreationData|.<code>[=clientDataJSONResult=]</code>.
+
+        :   {{AuthenticatorAttestationResponse/attestationObject}}
+        ::  |attestationObject|
+
+    :   {{PublicKeyCredential/clientExtensionResults}}
+    ::  A new {{AuthenticationExtensions}} object associated with |global| containing the bytes
```

Comment:
The record would just contain the value of extensionOutputsMap, not its bytes.

Reply:
ok, thx, so it seems i can just delete "the bytes of" ?

Reply:
i deleted "the bytes of".